packages/runtime-core/src/directives.ts

```diff
@@ -103,7 +103,7 @@ export function withDirectives<T extends VNode>(
         updated: dir
       } as ObjectDirective
     }
-    if (dir.deep) {
+    if (dir && dir.deep) {
```

Comment:
I think if `dir` is undefined, it should jump out of the current loop.

Reply:
Yes, we should really jump out of the loop. At present, such judgment and processing can also achieve the expected effect.

Reply:
I don't have complete reading logic. Thank you for reminding me.